SeaPkg/Library/BasePeCoffLibNegative/BasePeCoffLibNegative.c

```diff
@@ -1009,10 +997,25 @@ PeCoffImageDiffValidation (
         break;
     }
 
+    // Exit all validation checks if this validation entry fails
     if (EFI_ERROR (Status)) {
       break;
     }
 
+    // Exit all validation checks if this validation entry's default value overflows the Auxiliary file buffer.
+    if (ImageValidationEntryHdr->OffsetToDefault + ImageValidationEntryHdr->Size > ReferenceDataSize) {

```

Comment:
I think you want this check to occur earlier. We would not want to touch this region if this aux_bin is bad.

Reply:
Sure that makes sense. I'll move this check to before we run the symbol validation code

Reply:
moved to the top!